accessibility-checker/test/mocha/aChecker.Slow1/aChecker.ObjectStructure/aChecker.getCompliance.JSONObjectVerificationSelenium.test.js

```diff
@@ -180,7 +180,7 @@ describe("JSON Structure Verification Selenium", function () {
                                     })
                                     //console.log("report= " + JSON.stringify(report));
                                     // Run the diff algo to get the list of differences
-                                    differences = aChecker.diffResultsWithExpected(report, expected, false);
+                                    differences = aChecker.diffResultsWithExpected(report, expected, false);console.log("report= " + JSON.stringify(report));
```

Comment:
Please don't include more console.log's in PRs.

Reply:
changed